manifests/g/GottCode/FocusWriter/1.7.6/Gottcode.FocusWriter.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GottCode.FocusWriter
+PackageVersion: 1.7.6
+PackageLocale: en-US
+Publisher: Graeme Gott
+PublisherUrl: https://gottcode.org/
+PrivacyUrl: https://gottcode.org/privacy/
+Author: Graeme Gott
+PackageName: FocusWriter
+PackageUrl: https://gottcode.org/focuswriter/
+License: GPL-3.0-or-later
+LicenseUrl: https://www.gnu.org/licenses/gpl-3.0.html
+Copyright: Copyright (c) 2021 Graeme Gott. All rights are reserved.
+ShortDescription: Fullscreen, highly customizable text editor.
+Description: FocusWriter is a simple, distraction-free writing environment. It utilizes a hide-away interface that you access by moving your mouse to the edges of the screen, allowing the program to have a familiar look and feel to it while still getting out of the way so that you can immerse yourself in your work. It’s available for Linux and Windows, and has been translated into many different languages.
```

Comment:
```suggestion
Description: "FocusWriter is a simple, distraction-free writing environment. It utilizes a hide-away interface that you access by moving your mouse to the edges of the screen, allowing the program to have a familiar look and feel to it while still getting out of the way so that you can immerse yourself in your work. It’s available for Linux and Windows, and has been translated into many different languages."
```

Reply:
The last line contains "**It's** available"......  which is why manifest validation failed.

Reply:
![image](https://user-images.githubusercontent.com/15158490/127743393-a5db48d6-54aa-402f-a6b5-9cd8f05cfa80.png)

I don't see a reason for that, the `’` is a special character so it dosn't trigger as `'`